driver/random.hpp

```diff
@@ -1,32 +1,46 @@
 #ifndef GUARD_RANDOM_GEN_
 #define GUARD_RANDOM_GEN_
 
+#include <miopen/env.hpp>
+#include <iostream>
+#include <cstdlib>
 #include <random>
 
 std::minstd_rand& get_minstd_gen()
 {
-    static thread_local std::minstd_rand minstd_gen(std::random_device{}());
+    static thread_local std::minstd_rand minstd_gen{[]() {
+        auto external_seed = miopen::EnvvarValue("MIOPEN_DRIVER_PRNG_SEED", 100500);
+
+        auto seed = external_seed == 0
+                        ? std::random_device{}()
+                        : static_cast<std::random_device::result_type>(external_seed);
+        std::cout << "Random seed: " << seed << "\n";
```

Comment:
[R] The message gives an impression that that the seed is random, which is not always so. Maybe "PRNG seed"? or "Seed of random generator"?

Reply:
But it's actually random sometimes) Ok, I'll change it.